tools/kompos/src/messages.ts

```diff
@@ -115,12 +115,137 @@ export interface SectionBreakpointData extends AbstractBreakpointData {
   coord: FullSourceCoordinate;
 }
 
-export function createLineBreakpointData(sourceUri: string, line: number): LineBreakpointData {
-  return {
-    type: "LineBreakpoint",
-    line:      line,
-    sourceUri: sourceUri,
-    enabled:   false};
+export type Breakpoint = LineBreakpoint | MessageBreakpoint |
```

Comment:
It looks like this is a broken merge/rebase.
Please see the `breakpoints.ts` file next to `messages.ts`. This here seem to duplicate the code that moved there.